storage/migrations/1_chainwatch.go

```diff
@@ -21,15 +21,14 @@ CREATE EXTENSION IF NOT EXISTS timescaledb WITH SCHEMA public;
 
 CREATE TYPE public.miner_sector_event_type AS ENUM (
```

Comment:
So I've cheated a bit here since this is the first migration. I found it to be a bit of a headache to write the migration for this type and since we are ~burning~ not using the previous chainwatch data this feels relatively harmless, but I will write the migration if that's the desired outcome.

Reply:
Any existing sentinel database won't have the new enum values and you can't safely re-run the version 1 migration against them.

Maybe this migration would work:
```
ALTER TYPE public.miner_sector_event_type ADD VALUE 'PRECOMMIT_REMOVED' AFTER 'PRECOMMIT_ADDED';
ALTER TYPE public.miner_sector_event_type ADD VALUE 'SECTOR_REMOVED' AFTER 'SECTOR_ADDED';
```

Maybe a better solution is to create a new enum type and change the table to use it. That's what I'd do to keep backwards compatibility if this was a protobuf.